utils/wasm-builder/src/lib.rs

```diff
@@ -159,6 +164,25 @@ impl WasmBuilder {
         }
         Ok(matched_features)
     }
+
+    fn paths_to_remap(&self) -> Result<Vec<(PathBuf, &'static str)>> {
+        let home_dir = dirs::home_dir().context("unable to get home directory")?;
+
+        let project_dir = self.wasm_project.original_dir();
+
+        let cargo_dir = std::env::var_os("CARGO_HOME")
+            .map(PathBuf::from)
+            .context("unable to get cargo home directory")?;
+
+        let cargo_checkouts_dir = cargo_dir.join("git").join("checkouts");
+
+        Ok(vec![
+            (home_dir, "/home"),
+            (project_dir, "/proj"),
```

Comment:
Maybe?
```suggestion
            (project_dir, "/program"),
```

Reply:
Here we can also consider "code". I tried to give short remaps to reduce the size of the binary file and reduce fee for end-user.

Reply:
> Here we can also consider "code". I tried to give short remaps to reduce the size of the binary file and reduce fee for end-user.

I like "code" one